views/base.html

```diff
@@ -1,29 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
-   <head>
-       {% include 'meta.html' %}
-      <title>SoA</title>
-      <link rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css">
-      <style>
-        @import url('https://fonts.googleapis.com/css?family=Lato');
-        html { font-family: 'Lato', sans-serif; }
-        {% if alerts %}
-          .alert {
-          padding: 5px;
-          border: 1px solid #2c8898;
-          border-radius: 5px;
-          cursor:pointer;
-          }
-          .info{
-          background-color: #b7dfea;
-          }
-          .success{
-          background-color: #85ffc7;
-          }
-          .danger{
-          background-color: #f4989c;
-          }
-          @media only screen and (min-width: 768px) { 
+    <head>
+        {% include 'meta.html' %}
+        <title>SoA</title>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+        <link id="pagestyle" rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css" media="screen">
+        <!--
+            <script type="text/javascript">
+              function swapStyleSheet(sheet){
+                   document.getElementById('pagestyle').setAttribute('href',sheet);
+            
+              -->
+        <p>
+            Choose theme: 
+            <select>
+                <option selected="selected" value="sakura">Default</option>
+                <option value="sakura-dark">Night</option>
+                <option value="sakura-earthly">Earth</option>
+                <option value="sakura-vader">Vampire</option>
+            </select>
+        </p>
+        <script>
```

Comment:
Can we move this code to `themes.html` and use `{% include 'themes.html %}` in base? If possible I'd like to keep base as clean as possible.

Reply:
In account's Html file or theme.html?  both suggestions are different. or you want to just show it in account's HTML but using `{ % include 'themes.html' %}`